mozilla-release/browser/components/newtab/aboutNewTabService.js

```diff
@@ -179,6 +180,7 @@ AboutNewTabService.prototype = {
     // resource://activity-stream/prerendered/ar/activity-stream.html
     // resource://activity-stream/prerendered/en-US/activity-stream-prerendered.html
     // resource://activity-stream/prerendered/static/activity-stream-debug.html
+    return LOCAL_NEWTAB_URL;  // Cliqz. Default URL is NewTab page. Always.
```

Comment:
@alver-cliqz shouldn't we remove the next lines?

Reply:
Next lines from original Firefox code, I don't want to touch it to decrease number of possible merge conflicts